### DISCUSSION TITLE
L/G LGCIU 1+2 Fault

### DISCUSSION THREAD

**Discussion by mali6491** [flybywiresim/aircraft#7248] [Help] [answered]
Unless I am mistaken failures are not implemented in the A32NX .
I just got an ECAM fault indication shortly after a successful takeoff:
 L/G LGCIU 1+2 FAULT   
   -L/G  GRVTY EXTN

Noticed that gear lever had no effect and landing gears could not be retracted.
I didnt find any abnormal indications on the overhead or anywhere else.

Unfortunately I was not able to complete the flight and had to exit the sim.

**Comment by tracernz** [comment, marked as answer]
Some failures are implemented. These particular ones are not "failures" as such, but warnings that can be triggered by certain things in-flight e.g. gear extended at too high speed causing "erroneous" values in the LGCIUs, as in the real aircraft.
